doc/CONFIGURATION.md

```diff
@@ -83,8 +86,18 @@ Here is an example least-privilege policy document to add to an IAM user or role
    ]
 }
 ```
+Mountpoint also respects access control lists (ACLs) applied to objects in your S3 bucket, but does not allow you to automatically attach ACLs to objects created with Mountpoint. A majority of modern use cases in Amazon S3 no longer require the use of ACLs. We recommend that you keep ACLs disabled for your S3 bucket, and instead use bucket policies to control access to your objects.
 
-Directory buckets, introduced with the S3 Express One Zone storage class, use a different authentication mechanism from general purpose buckets. Instead of using `s3:*` actions, you should allow the `s3express:CreateSession` action. Here is an example of least-privilege policy document.
+### Directory buckets
+Directory buckets, introduced with the S3 Express One Zone storage class, use a different authentication mechanism from general purpose buckets.
```

Comment:
I think we should introduce XOZ in relation to the session-based auth, not directory buckets, since we now have Z-IA.